### PR TITLE
add rooms and maincolors to suggestion page

### PR DESCRIPTION
**Pull request by NilsOt1**
I didn't create the `span` component (US 3) because I can use my `MainColor` Component to display the colors on the `suggestion page`. I disabled the color picker for the `suggestion page`

### CODE REVIEW

File: components/Rooms/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export default function Rooms() {
+  return (
+    <>
+      <label htmlFor="rooms">Rooms</label>
+      <select name="rooms" id="rooms">
```
Comment (julialoeschel) [inline components/Rooms/index.js:5]: If there is only one option, I guess a select is not necessary jet :)

**Review by julialoeschel — COMMENTED** [review]
great job! 

I only found a tiny issue, which I think will resolve soon anyway! 

keep on going!

File: components/Rooms/index.js (hunk 1 above)
Comment (NilsOt1) [reply components/Rooms/index.js]: Thanks for the hint. I replaced the `select` with a `span` for now and implement the select in a later US

**Comment by Dariastep** [comment]
LGTM! Keep going

**Pull request merged**